requirements.txt

```diff
@@ -121,7 +121,8 @@ invenio-config==1.0.1
 invenio-csl-rest==1.0.0a1
 invenio-db==1.0.3
 invenio-deposit==1.0.0a9
-invenio-files-rest==1.0.0a23.post1
+-e git+https://github.com/slint/invenio-files-rest.git@webpack-post2#egg=invenio-files-rest
```

Comment:
I think I rebased on a branch I was working... If possible cherry-pick your commits and ignore the `invenio-files-rest`-related commits I did.